### PR TITLE
✨ Adding Navigation, Contact Info, and UI Refinements to Awesome Books Website

### PR DESCRIPTION
**Pull request by DevKaranJ**
**This pull request ✨ adds significant enhancements to the Awesome Books website project, incorporating a navigation bar 🚀, a new "Contact Info" section 📞, and refining the user interface for a more seamless experience. The changes align with the project's learning objectives of understanding single page applications 📚, manipulating DOM elements using JavaScript 🎨, and working with JavaScript events 🎉.**

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -5,8 +5,22 @@ class BookManager {
...
+
+    document.querySelector('.item1 a').addEventListener('click', () => {
+      this.showContent(this.listContent);
+    });
+    document.querySelector('.item2 a').addEventListener('click', () => {
+      this.showContent(this.addNewContent);
+    });
+    document.querySelector('.item3 a').addEventListener('click', () => {
+      this.showContent(this.contactContent);
+    });
```
Comment (Olamarx) [inline script.js:23]: - [ ] When I refresh the list of books, it shows the form which should not be, I will encourage you to double-check the logic.
image

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Linters passed ✔️
- Readme file professional ✔️
- Three pages ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js (hunk 1 above)
Comment (richie1988) [reply script.js:23]: Please try to explain in a more understandable way which logic try to give us the information of the logic cause you just said logic not explaining the main focus of the problem.

**Review by skyv26 — APPROVED** [review]
## Hi @DevKaranJ, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

## STATUS: APPROVED 🟢 



## Highlights
- Good **PR** title ✅ 
- Descriptive *PR* description ✅ 
- Implement the features ✅ 
- Nice *UI* design ✅ 
- Professional `README` template ✅ 
- Linters are green ✅ 
- Correct `GitHub` workflow ✅ 
- Meaningful **Github** commits ✅ 
- Correct `.github/workflows` for linters ✅ 
- Followed *best-practices* ✅ 
- Correct **Semantics** used ✅ 
- Well-structured project ✅ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## ⚠️  WARNING ⚠️ 
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

## Have any doubt ❓ 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @skyv26 in your question so I can receive the notification. You can also follow me on the below platforms

<table>
<tr>
<td>

<a href="https://github.com/skyv26">
   @skyv26
</a>

</td>
<td>

<a href="https://www.linkedin.com/in/skyv2022/">
   linkedin-skyv2022
</a>

</td>
<td>

<a href="https://twitter.com/vrma_aakash">
   @vrma_aakash
</a>

</td>
<td>

<a href="https://www.instagram.com/skybrel/">
   @skybrel
</a>

</td>
<td>

<a href="https://www.frontendmentor.io/profile/skyv26">
   @skybrel
</a>

</td>
</tr>
</table>

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**